frontend/packages/knative-plugin/src/components/add/event-sources/EventSourcesSelector.scss

```diff
@@ -0,0 +1,17 @@
+.odc-event-source-filter {
+  display: flex;
+  align-items: center;
+
+  &__bar {
+    max-width: 20em;
+  }
+  &__items-count {
+    color: var(--pf-global--Color--200);
+    margin-left: auto;
+  }
+}
+
+.odc-event-source-selector {
+  height: 260px;
```

Comment:
This adds too much white beneath the selector when there is a single row.
Perhaps max-height is better suited here.

Reply:
Updated.